testing-guidance/workflow-tests/physics/physics-ragdoll-workflows.md

```diff
@@ -33,14 +34,14 @@ Any project that has the following Gems enabled:
+| **Manipulators** | <ol><li>In the Animation Editor open the Action History, by going to View → Action History. Place the window in a visible and convenient place (for example dock it under the Skeleton Outliner pane).</li> <li>Modify any bone using any of the manipulators in the Atom Renderer Window (same as in the *Verify Atom Render Window UI clusters* workflow).</li> <li>Modify any bone by changing properties related to the button clusters under the property componenets in the Inspector pane (for example *Swing limit Y*, or *Twist upper limit*).</li> <li>Perform Undo/Redo actions on changes made by using Atom Render Window, and modifying properties under the Ragdoll pane. Undo/Redo actions can be performed by using Ctrl+Z/Ctrl+Shift+Z shortcuts, or by selecting Edit → Undo/Edit → Redo.</li></ol>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           | <ul><li>Action History can be opened.</li><li>Modifications made in the Atom Render Window using the manipulators is saved to the Action History and is affected by Undo/Redo actions.</li><li>Modifications made in the Ragdoll pane by adjusting properties is saved to the Action History and is affected by Undo/Redo actions.</li></ul>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
```

Comment:
* `under the property componenets` - Should be `under the property components`.

Reply:
Fixed